### PR TITLE
getRowIdViaUniqueId method

### PR DESCRIPTION
**Pull request by Avivbh**
New method that returns row id based on unique id provided to it.
If the unique id cant be found in the table it will return "-1".

The idea behind the new method is that there are some method right now that don't support unique id (like updateCell, expandRow, collapseRow...), with getRowIdViaUniqueId  you will be able to fetch the row id before calling other methods.

### CODE REVIEW

**Comment by djhvscf** [comment]
Please update the documentation also, @Avivbh

**Comment by Avivbh** [comment]
Done :) @djhvscf

**Comment by sketchman-nyt** [comment]
Would changing the name from 'getRowIdViaUniqueId' to 'getRowIdByUniqueId' better for consistency?

**Comment by djhvscf** [comment]
@sketchman-nyt I completely agree with that.. @wenzhixin

**Comment by Avivbh** [comment]
@sketchman-nyt  That's a good idea, changed the method name.

**Comment by djhvscf** [comment]
@Avivbh  great! :+1:

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -2203,6 +2203,12 @@
         return dataRow;
     };
 
+    BootstrapTable.prototype.getRowIdByUniqueId = function(uniqueId){
+        var rowId;
+        rowId = $.inArray(this.getRowByUniqueId(uniqueId), this.options.data);
+        return rowId;
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: `return $.inArray(this.getRowByUniqueId(uniqueId), this.options.data);`

File: docs/_i18n/en/documentation/methods.md [hunk 2]
```diff
@@ -122,7 +122,13 @@ The calling method syntax: `$('#table').bootstrapTable('method', parameter);`.
         row: the new row data.
         </td>
     </tr>
-	<tr>
+    <tr>
+        <td>getRowIdByUniqueId</td>
+        <td>id</td>
```
Comment (wenzhixin) [inline docs/_i18n/en/documentation/methods.md]: uniqueId

**Comment by djhvscf** [comment]
@Avivbh any update here?

**Comment by wenzhixin** [comment]
I think we can use:
```
$.inArray($table.bootstrapTable('getRowByUniqueId', uniqueId), $table.bootstrapTable('getData'));
```
instead, thanks anyway.

**Pull request closed without merging**